### PR TITLE
Copy GTEx whole blood association cis-QTLs

### PR DESCRIPTION
**Pull request by KatalinaBobowik**
Copy v8 GTEx whole blood association cis-QTLs from gtex-resources GCP bucket to `main`.

### CODE REVIEW

File: data_transfer/copy_gtex_v8_whole_blood_data.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+set -ex
+
+gsutil -u tx-adapt -m cp "gs://gtex-resources/GTEx_Analysis_v8_QTLs/GTEx_Analysis_v8_EUR_eQTL_all_associations/Whole_Blood.v8.EUR.allpairs.*" gs://cpg-gtex-main/v8/whole_blood/
```
Comment (lgruen) [inline data_transfer/copy_gtex_v8_whole_blood_data.sh:5]: Did you also want to copy a subset of the data (e.g. chr22) to the `test` bucket?

**Review by lgruen — CHANGES_REQUESTED** [review]

File: data_transfer/copy_gtex_v8_whole_blood_data.sh (hunk 1 above)
Comment (KatalinaBobowik) [reply data_transfer/copy_gtex_v8_whole_blood_data.sh:5]: Good catch. Yes, I've now added that in.

File: data_transfer/copy_gtex_v8_whole_blood_data.sh [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+gsutil -u tx-adapt -m cp gs://gtex-resources/GTEx_Analysis_v8_QTLs/GTEx_Analysis_v8_EUR_eQTL_all_associations/Whole_Blood.v8.EUR.allpairs.chr22.parquet gs://cpg-gtex-test/v8/whole_blood/
```
Comment (lgruen) [inline data_transfer/copy_gtex_v8_whole_blood_data.sh]: It won't make a huge difference, but if you copy from our `main` bucket as the source, we won't have to pay for egress twice.

**Review by lgruen — APPROVED** [review]

File: data_transfer/copy_gtex_v8_whole_blood_data.sh (hunk 2 above)
Comment (KatalinaBobowik) [reply data_transfer/copy_gtex_v8_whole_blood_data.sh]: I can do that! Is there a way to ensure the first command finishes before the second is executed?

Comment (lgruen) [reply data_transfer/copy_gtex_v8_whole_blood_data.sh]: This happens automatically: every line is executed sequentially in a bash script and `gsutil` only returns once the copy is complete. If you're worried about the first call failing, you've also already taken care of that with `set -ex`, which will make the whole script abort and fail as soon as any command returns a status error code.

**Review by lgruen — APPROVED** [review]

File: data_transfer/copy_gtex_v8_whole_blood_data.sh (hunk 2 above)
Comment (KatalinaBobowik) [reply data_transfer/copy_gtex_v8_whole_blood_data.sh]: great, that makes sense. Thanks!

**Pull request merged**